index.html

```diff
@@ -29,9 +29,120 @@ <h1>Felipe<br>Oyarzo</h1>
                 </p>
             </div>
             <button class="button" type="button" id="btnHeadline">Start collaboration</button>
-            <div class="geometric"></div>            
+            <div class="geometric"></div>
         </section>
     </main>
+    <section class="projects">
+        <h1>Projects</h1>
+        <div class="grid-container">
+            <div class="card card1">
+                <img src="./assets/img/projects-img1.png" alt="Screenshot of project 1">
+                <div class="card-body">
+                    <h4>Project name goes here</h4>
+                    <ul class="card-list">
+                        <li class="card-list-item">HTML/CSS</li>
+                        <li class="card-list-item">Ruby on Rails</li>
+                        <li class="card-list-item">JavaScript</li>
+                    </ul>
+                    <button type="button" class="card-btn">See this project &rarr;</button>
+                </div>
+            </div>
+            <div class="card card2">
+                <img src="./assets/img/projects-img2.png" alt="Screenshot of project 1">
+                <div class="card-body">
+                    <h4>Project name goes here</h4>
+                    <ul class="card-list">
+                        <li class="card-list-item">HTML/CSS</li>
+                        <li class="card-list-item">Ruby on Rails</li>
+                        <li class="card-list-item">JavaScript</li>
+                    </ul>
+                    <button type="button" class="card-btn">See this project &rarr;</button>
+                </div>
+            </div>
+            <div class="card card3">
+                <img src="./assets/img/projects-img3.png" alt="Screenshot of project 1">
+                <div class="card-body">
+                    <h4>Project name goes here</h4>
```

Comment:
- [ ] Kindly use `<p>` tag for this element instead of `<h4>`

Reply:
Good job following the advice of the previous reviewer.